### ISSUE TITLE
Incorrect option name file_type used in example

### ISSUE THREAD

**Issue by bersbersbers** [r-lib/styler#854]
https://github.com/r-lib/styler/blob/e7777f259d320413d51af36da383024f97630a9f/R/ui-styling.R#L241-L245

https://github.com/r-lib/styler/blob/e7777f259d320413d51af36da383024f97630a9f/R/ui-styling.R#L232

**Comment by lorenzwalthert** [comment]
Right... Thanks @bersbersbers. Want to fix it with a PR (and run `roxygen2::roxygenize()`)? I will do it otherwise...
